src/Controller/NotifyController.php

```diff
@@ -33,14 +42,14 @@ public function verifyImojeNotification(Request $request): Response
             $gateway->execute(new Notify($notifyToken));
 
             return new JsonResponse(['status' => 'ok']);
-        } else {
-            throw new NotFoundHttpException('Payment token not found');
         }
+
+        throw new NotFoundHttpException('Payment token not found');
```

Comment:
i think you can reverse the if condition. I mean if $request === null then throw exception and the rest of logic outside the if. It is name `Return early pattern` you can read about this.